src/mscorlib/shared/System/IO/PathHelper.Windows.cs

```diff
@@ -50,12 +50,6 @@ internal static string Normalize(string path, bool checkInvalidCharacters, bool
                 // Trim whitespace off the end of the string. Win32 normalization trims only U+0020.
                 fullPath.TrimEnd(PathInternal.s_trimEndChars);
 
-                if (fullPath.Length >= PathInternal.MaxLongPath)
```

Comment:
Delete definitions of MaxLongPath and MaxComponentLength since they are not longer used?